### PR TITLE
Add: new  article "is-the-developer-community-useful?"

### PR DESCRIPTION
**Pull request by SHABA-Rex**
Hi everyone here is a new article on the importance of a developer being in a community.
Thank you in advance for your reviews and feedback. 😃

### CODE REVIEW

File: content/post/articles/is-the-developer-community-useful.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+categories: ["Software development"]
+tags: ["Coding","Community","Osscameroon","Developers","CaParleDev"]
+authors: ["Shaba-Rex"]
+---
+
+{{<figure src="/is-developer-community-useful/osslogo.png"  alt="Open source community">}}
+
+An open-source community is a set of contributors from all over the world who share an interest in meeting a common need, starting from small projects to large ones, typically software projects. They carry them out in a high-performance collaborative development environment, allowing the organizational scheme and processes to emerge over time.
+
+A developer community is a group of developers gathered to achieve a common goal. They come together to share their knowledge, improve the skills of participants. The developer community is an OSC, it is one of its many forms(developer communities, user communities,…). The answer to the question ‘*is the developer community useful?*’ is obviously yes in this article, it will not be for us to discuss the usefulness of an OSC(in our case, it is a developer community), but rather to demonstrate it through two main questions:
```
[the remark this answers is not in the document]
Comment (SHABA-Rex) [reply content/post/articles/is-the-developer-community-useful.md:14]: Thank you for your review 😺, OSC means Open Source Community the definition of this term is on the 12th line, and the definition of the acronym is on the 19th line.

I don't know why but I like to separate the acronyms from their definition, for me, it's more readable like that.

[the remark this answers is not in the document]
Comment (elhmn) [reply content/post/articles/is-the-developer-community-useful.md:14]: If you plan to use acronyms later in your text, it is nicer for the reader to add the acronym next to the definition the first time you mention it. Can you please update it ?

[the remark this answers is not in the document]
Comment (SHABA-Rex) [reply content/post/articles/is-the-developer-community-useful.md:14]: Good point, I will do it. 👍

[the remark this answers is not in the document]
Comment (SHABA-Rex) [reply content/post/articles/is-the-developer-community-useful.md:14]: Done👍 .

**Review by elhmn — APPROVED** [review]
Great article thanks 🥇

**Pull request merged**